src/Server.php

```diff
@@ -206,7 +241,18 @@ public function close()
         }
 
         $this->loop->removeStream($this->master);
+
+        //get socket meta_data before closing
+        $metaData = stream_get_meta_data($this->master);
+        $socket_path = stream_socket_get_name($this->master, false);
         fclose($this->master);
+
+        //if unix socket we must delete socket file
```

Comment:
We need to delete socket file, because the socket file will remain after server close.